arduino/smartcar/smartcar.ino

```diff
@@ -126,33 +147,24 @@ void setup()
     mqtt.subscribe("/smartcar/control/#", 1);
     mqtt.subscribe("/smartcar/safetysystem", 1);
     mqtt.onMessage([](String topic, String message)
-                   {
-     //this is going to be commented out because it's going to appear on the android side but the code remains in case someone needs it for reference.
-//         //if statements controls the intake of information from the joystick forward and backwards
-//         speedGate = message.toInt();
-//         if (driveForwards && speedGate >= 0){
-//             theSpeed = speedGate;
-//         } 
-//         if (drivebackwards && speedGate <= 0){
-//             theSpeed = speedGate;
-//         }
-//             //Serial.println(theSpeed);//shows the speed
-        
+    {
         if (topic == "/smartcar/control/throttle") {
-            Serial.println(message);
-            car.setSpeed(message.toInt());//this needs to be swapped if the above is uncommented <<car.setSpeed(theSpeed);>>
+            Serial.println(message);//Liam are these needed
+            car.setSpeed(message.toInt());
         } else if (topic == "/smartcar/control/steering") {
-            Serial.println(message);
+            Serial.println(message);//Liam are these needed
```

Comment:
Same as above.

Reply:
Same as before is it needed the Serial.println(message);